test/e2e/playbook/nri-balloons-plugin-deploy.yaml

```diff
@@ -65,9 +65,8 @@
       when: is_containerd
 
     - name: load nri plugin image when using cri-o
-      become: yes
+      become: no
       shell: "{{ item }}"
```

Comment:
L71 runs as root, don't we need a sudo privilege (`become: yes`)  for that ?

Reply:
No. That would run the command as root, not just assign you a sudo priviledge. The original playbook was trying to do that: `podman image load` with `become: yes` and then do another `podman image scp` to make the image visible to the CRI runtime. But that does not work (I think it would be possible to get it working with an extra reboot). It is simpler/faster to do the import with a single command. But to do that you literally need to do a `sudo podman image load` instead of doing a `podman image load` as root, which documentedly behaves differently.